docs/source/advanced/sagemaker-pipelines.rst

```diff
@@ -126,14 +126,22 @@ We list those here, with the step name that you can provide in ``--jobs-to-run``
    Multi-instance graph partitioning for distributed training.
 
 4. **GraphBolt Conversion** (``gb_convert``):
-   Converts partitioned data to GraphBolt format for improved performance.
+   Converts partitioned data to GraphBolt format for improved training/inference efficiency..
 
 5. **Training** (``train``):
    Trains your graph neural network model.
 
 6. **Inference** (``inference``):
    Runs predictions using your trained model.
 
+The choice of jobs to run will mostly stem from the size of your graph.
+For graphs that can fit into the memory of one machine, a typical
+job sequence would be ``gconstruct train inference``.
+
+For graphs that are too large to fit into one machine, you will need to
+pre-process them using GSProcessing and use distributed GSPartition.
+Such a sequence of jobs would be ``gsprocessing dist_part train inference``.
```

Comment:
Do we support `gsprocessing dist_part inference` ?

Reply:
I haven't tried it but we don't have any restrictions against such a sequence.

Reply:
Actually took a look at the implementation. Right now we do not support this because the inference step always gets its model S3 path from a train step which we assume was created previously. 

To support inference-only pipeline creation we would need to make a couple of changes to the pipeline creation code.
In code a user can hack this by assigning a value to `self.model_input_path` after they create a `GraphStormPipelineGenerator`, but there's no CLI support yet.

Reply:
Let's create a corresponding issue for 0.5 release.

Reply:
Created https://github.com/awslabs/graphstorm/issues/1211